integrations/lib/multiwoven/integrations/destination/zendesk/config/spec.json

```diff
@@ -0,0 +1,25 @@
+{
+  "documentation_url": "https://docs.multiwoven.com/destinations/customer-support/zendesk",
+  "stream_type": "static",
+  "connection_specification": {
+    "$schema": "http://json-schema.org/draft-07/schema#",
+    "title": "Zendesk",
+    "type": "object",
+    "required": [
+      "username",
+      "password"
+    ],
+    "properties": {
+      "username": {
+        "type": "string",
+        "title": "Username",
+        "order": 0
+      },
+      "password": {
+        "type": "string",
+        "title": "Password",
+        "order": 1
```

Comment:
Please add `multiwoven_secret: true` for masking the password in frontend

Reply:
done. committed.